packages/dev/cypress/e2e/unovis.cy.ts

```diff
@@ -0,0 +1,358 @@
+    cy.wait(3000)
+    cy.percySnapshot('Custom Style Brush', { scope: '.exampleViewer' })
+  })
+
+  it('Stacked vs Non-Stacked', () => {
+    cy.contains('Stacked vs Non-Stacked').click()
+    cy.wait(3000)
+    cy.percySnapshot('Stacked vs Non-Stacked', { scope: '.exampleViewer' })
+  })
+
+  it('Dual Axis Chart', () => {
+    cy.contains('Dual Axis Chart').click()
+    cy.wait(3000)
+    cy.percySnapshot('Dual Axis Chart', { scope: '.exampleViewer' })
+  })
+
+  it('Basic Grouped Bar Chart', () => {
+    cy.contains('Basic Grouped Bar Chart').click()
+    cy.wait(3000)
+    cy.percySnapshot('Basic Grouped Bar Chart', { scope: '.exampleViewer' })
+  })
+
+  it('Grouped Bar Chart', () => {
+    cy.contains('Grouped Bar Chart').click()
+    cy.wait(3000)
+    cy.percySnapshot('Grouped Bar Chart', { scope: '.exampleViewer' })
+  })
+
+  it('Multi Line Chart', () => {
+    cy.contains('Multi Line Chart').click()
+    cy.wait(3000)
+    cy.percySnapshot('Multi Line Chart', { scope: '.exampleViewer' })
+  })
+
+  it('Points with labels', () => {
+    cy.contains('Points with labels').click()
+    cy.wait(3000)
+    cy.percySnapshot('Points with labels', { scope: '.exampleViewer' })
+  })
+
+  it('Points with stroke', () => {
+    cy.contains('Points with stroke').click()
+    cy.wait(3000)
+    cy.percySnapshot('Points with stroke', { scope: '.exampleViewer' })
+  })
+
+  it('Scatter with Line', () => {
+    cy.contains('Scatter with Line').click()
+    cy.wait(3000)
+    cy.percySnapshot('Scatter with Line', { scope: '.exampleViewer' })
+  })
+
+  it('Basic Stacked Bar Chart', () => {
+    cy.contains('Basic Stacked Bar Chart').click()
+    cy.wait(3000)
+    cy.percySnapshot('Basic Stacked Bar Chart', { scope: '.exampleViewer' })
+  })
+
+  it('Stacked Bar Data Transitions', () => {
+    cy.contains('Stacked Bar Data Transitions').click()
+    cy.wait(3000)
+    cy.percySnapshot('Stacked Bar Data Transitions', { scope: '.exampleViewer' })
+  })
+
+  it('Timeline Data Transitions', () => {
+    cy.contains('Timeline Data Transitions').click()
+    cy.wait(3000)
+    cy.percySnapshot('Timeline Data Transitions', { scope: '.exampleViewer' })
+  })
+
+  it('Timeline: Negative Lengths', () => {
+    cy.contains('Timeline: Negative Lengths').click()
+    cy.wait(3000)
+    cy.percySnapshot('Timeline: Negative Lengths', { scope: '.exampleViewer' })
+  })
+
+  it('Tooltip and Scrolling', () => {
+    cy.contains('Tooltip and Scrolling').click()
+    cy.wait(3000)
+    cy.percySnapshot('Tooltip and Scrolling', { scope: '.exampleViewer' })
+  })
+
+  it('Scale by Domain', () => {
+    cy.contains('Scale by Domain').click()
+    cy.wait(3000)
+    cy.percySnapshot('Scale by Domain', { scope: '.exampleViewer' })
+  })
+
+  it('Stacked Bar with Labels', () => {
+    cy.contains('Stacked Bar with Labels').click()
+    cy.wait(3000)
+    cy.percySnapshot('Stacked Bar with Labels', { scope: '.exampleViewer' })
+  })
+})
+
```

Comment:
We can try generating this file automatically based on the example files. Technically they have the required information, like the description text.

Reply:
> We can try generating this file automatically based on the example files. Technically they have the required information, like the description text.

I suppose we could, but some page require additional settings. For example the `API Endpoints Tree` requires a bigger minHeight, we'd have to manually set things like that for specific test cases. Do you have a workaround for cases like this?

Reply:
You can have a map of additional settings in the testing file to apply on the go if needed. I can explain more if you want, just let me know.

Reply:
Do you mean to have an array or load a separate file which contains 
```ts
testingArray = [
  {name: 'Single Container', duration:3000, additionalParams: {width: [1300]}},
  {name: 'Sankey', duration:3000},  ...
]
```
And iterate through the array inside the `unovis.cy.ts`?

Reply:
Take a look at `unovis/packages/dev/src/examples/index.ts`, I was thinking about something similar to generate this test file automatically. You'll need to "compile" it using Webpack but I think it will be a reasonable trade-off.

So the script will:
- Batch import all the `index.tsx` files in the examples folder;
- You'll be able to get the required information from the imports (titles, ...);
- You can define additional testing configuration in the `unovis.cy.ts` file and match the corresponding examples by name or path. Or you can define it in the `index.tsx` files of the examples themsevles.

Reply:
@rokotyan I will give it a try. Is this PR getting too big? Should we have a separate PR for this and the `duration` params?

Reply:
Up to you. I would do it here, but I don't see any problem with merging this PR first and having another one